### ISSUE TITLE
api.GetFriends is wrongfully requiring authentication

### ISSUE THREAD

**Issue by GoogleCodeExporter** [gyc2432968/python-twitter#88]
```
What steps will reproduce the problem?
1. api = twitter.Api() # unauthenticated
2. friends = api.GetFriends(user="spam")
3.

What is the expected output? What do you see instead?
I expect to get the friends list.  I get an error saying that the API instance 
needs to be 
authenticated.

What version of the product are you using? On what operating system?
python_twitter-0.6-py2.6.egg

Please provide any additional information below.
According to the twitter API documentation, using either friendships/show or 
statuses/friends 
does not require authentication.

```

Original issue reported on code.google.com by `jerry.he...@gmail.com` on 5 Aug 2009 at 6:05

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r183.
```

Original comment by `dclinton` on 5 Aug 2009 at 6:15
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report.  Fixed in r183.  Please verify.
```

Original comment by `dclinton` on 5 Aug 2009 at 6:15

**Comment by GoogleCodeExporter** [comment]
```
Confirm fixed with r183.
```

Original comment by `jerry.he...@gmail.com` on 8 Aug 2009 at 4:30

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision b3200e4d6f.
```

Original comment by `dclinton` on 24 Oct 2009 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision b83d4438e3.
```

Original comment by `dclinton` on 24 Oct 2009 at 4:58
